### PR TITLE
fix lang attribute

### PR DESCRIPTION
**Pull request by masuP9**

### CODE REVIEW

**Comment by masuP9** [comment]
コミットメッセージでタイポ

File: src/parenting-who/index.njk [hunk 1]
```diff
@@ -27,7 +27,9 @@ date: Last Modified
     <script async src="https://www.googletagmanager.com/gtag/js?id=UA-162419995-1"></script>
     <script>
       window.dataLayer = window.dataLayer || [];
-      function gtag(){dataLayer.push(arguments);}
+      function gtag() {
+        dataLayer.push(arguments);
+      }
```
Comment (masuP9) [inline src/parenting-who/index.njk:32]: ヌー

**Comment by oti** [comment]
@masuP9 
```
<i class="original-document-title">Coronavirus disease (COVID-19) advice for the public: Healthy Parenting</i>
```
ここも `lang="en"` にしたほうがいいかも。かつ、ここは斜体のままが望ましいです。

File: src/parenting-who/index.njk (hunk 1 above)
Comment (oti) [reply src/parenting-who/index.njk:32]: まぁ、いいんじゃないかな…

Comment (masuP9) [reply src/parenting-who/index.njk]: WorkSpaceの設定で Format On Save を切った

**Comment by masuP9** [comment]
そこは親の `<p>` を `lang="en"` にしてるからいいかなーって。あとどっちかというと `cite` 要素かも。 `cite` は引用元じゃなくても使えるっぽいし。

> The cite element represents the title of a work (e.g. a book, a paper, an essay, a poem, a score, a song, a script, a film, a TV show, a game, a sculpture, a painting, a theatre production, a play, an opera, a musical, an exhibition, a legal case report, a computer program, etc). This can be a work that is being quoted or referenced in detail (i.e. a citation), or it can just be a work that is mentioned in passing.

https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-cite-element

**Pull request merged**